Backpack/build.gradle

```diff
@@ -64,6 +64,8 @@ dependencies {
   androidTestImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:$rootProject.ext.mockitoKotlin"
   androidTestImplementation "org.mockito:mockito-android:$rootProject.ext.mockitoAndroid"
   testImplementation "junit:junit:$rootProject.ext.junit"
+
+  lintPublish project(':backpack-lint')
```

Comment:
this publishes the lint checks with backpack, so when you integrate backpack the lint checks are added

Reply:
Does this mean anyone using Backpack Android does not need to worry about enabling the rules and if you are using the library 'it just works'?

Reply:
correct - as long as they use lint the checks will run